dashboard/src/utils/helper.js

```diff
@@ -5,6 +5,13 @@ export const uid = () => {
   return head + tail;
 };
 
+/**
+ * Check if password entered by user is valid or not
```

Comment:
It would be good to provide a least a little more here.  For instance, "Determine whether the specified password meets the requirements for a valid password, including length, presence of upper-case letters, digits, and special characters."